mentor_classifier/mentor_classifier/api.py

```diff
@@ -167,6 +202,30 @@ def fetch_mentor_data(mentor: str) -> dict:
     return data
 
 
+def fetch_category(category: str) -> dict:
+    tdjson = __auth_gql(query_category_answers(category))
+    return tdjson.get("data") or {}
+
+
+def generate_followups(
+    category: str, shared_root=SHARED_ROOT
```

Comment:
get rid of the `shared_root` arg here and just use (new func) `get_shared_root` inside this function